modules/express/src/config.ts

```diff
@@ -83,18 +83,20 @@ export const DefaultConfig: Config = {
 /**
  * Helper function to merge config sources into a single config object.
  *
- * Earlier configs have higher precedence over subsequent configs.
+ * Later configs have higher precedence over earlier configs.
  */
 function mergeConfigs(...configs: Partial<Config>[]): Config {
   function isNilOrNaN(val: unknown): val is null | undefined | number {
     return isNil(val) || (isNumber(val) && isNaN(val));
   }
+
   // helper to get the first defined value for a given config key
   // from the config sources in a type safe manner
```

Comment:
I _think_ this comment is accurate, since we reversed the args and nix'd the `.reverse`, but wanted to call your attention to it in case it needs to be updated

Reply:
Good catch, this actually produces the last defined value.